recipes/nand2tetris-core

```diff
@@ -0,0 +1,3 @@
+(nand2tetris-core :fetcher github
+                  :repo "CestDiego/nand2tetris.el"
+                  :files ("nand2tetris-core.el"))
```

Comment:
You don't need package requirements between files in the same package. So put all the related files in one package, and then they can just `require` each other. The package metadata would go in the "main" `.el` file - the one with the name matching the package name. Hope that helps.